docs/pages/api-reference/operators.md

```diff
@@ -236,4 +236,25 @@ The `assign` operator has the following parameters:
 Fennel does not allow you to assign keys or timestamp columns from a dataset.
 :::
 
+### Groupby/ Window
 
+Fennel allows you to groupby and get a stream of windowed events for each group using the `window` operator.
+A great example of windowed events are sessions, where in we create window of events occurring together.
```

Comment:
In Fennel, the `window` operator enables grouping events into windowed streams based on specified criteria. A common application of this operator is in creating sessions, where it forms a window comprising events that occur in a session.
The `window` operator must be preceded by `groupby` to identify the fields that will be used for grouping the events before they are windowed.

Reply:
made the change.